old/fennel.lua

```diff
@@ -2693,7 +2693,7 @@ local specials = (function()
             -- Compile the forms into subChunk; compiler.compile1 is necessary for all nested
             -- includes to be emitted in the same root chunk in the top-level module
             for i = 1, #forms do
-                local subopts = i == #forms and {nval=1, tail=true} or {}
+                local subopts = i == #forms and {nval=1, tail=true} or {nval=0}
```

Comment:
This one is actually a legitimate bug in the old compiler that we didn't find until we tried to use `include` on Fennel itself.

Reply:
Nicely spotted!